trilium-note/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  server:
+    image: zadam/trilium:latest
+    restart: on-failure
+    hostname: "${DEVICE_HOSTNAME}"
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - VERSION=docker
+      - TRILIUM_DATA_DIR=/home/node/trilium-data
+    ports:
+      - 8080:8080
+    volumes:
+      - ${UMBREL_ROOT}/data/storage/trilium:/home/node/trilium-data
+    networks:
```

Comment:
We no longer require apps to define container IPs, so this section can be dropped.

Reply:
But I tested it quite a few times and the app was going into a different location sometimes when I removed this. So I tried to put it here as it was then working fine almost all of the times.

Reply:
I'm not sure what you're referring re 'going into a different location'. What I mean is; this `networks` block can be removed because we longer need to set container IPs and this simplifies deploying apps on Umbrel. You'll see many apps are still doing this, but this was before we stopped adding this section.

Reply:
Ok cool. I followed the default template from the umbrel docs that's why it's there. I will remove and push the changes. Thanks

Reply:
I have made the necessary changes and pushed the commit.